app/views/examples/confirmation.html

```diff
@@ -9,10 +9,6 @@
   <div class="grid-row">
     <div class="column-two-thirds">
 
-      <h1 class="heading-xlarge">
-        Example service name
-      </h1>
-
       <div class="govuk-box-highlight">
         <h2 class="bold-large">Application complete</h2>
```

Comment:
Maybe "Application complete" should have an h1 tag - as that's the main point of this screen, and every page should have an h1. It can stay the same font size, that's fine.